src/Monolog/Handler/ErrorLogHandler.php

```diff
@@ -71,13 +71,14 @@ protected function getDefaultFormatter(): FormatterInterface
      */
     protected function write(array $record)
     {
-        if ($this->expandNewlines) {
-            $lines = preg_split('{[\r\n]+}', (string) $record['formatted']);
-            foreach ($lines as $line) {
-                error_log($line, $this->messageType);
-            }
-        } else {
+        if (!$this->expandNewlines) {
```

Comment:
This makes the code a bit easier to read and removes the else (which is not needed at all here).

The other way would put the whole function in an additional un-needed code block.